jplag.frontend.scala/src/test/resources/de/jplag/scala/Parser.scala

```diff
@@ -0,0 +1,376 @@
+package de.jplag.scala
```

Comment:
Why is the parser a resource ?

Reply:
I used a duplicate of the parser file as a test file :)